tiledb/core.cc

```diff
@@ -1681,6 +1681,26 @@ py::str as_built_dump() {
   return res;
 }
 
+auto walk_callback = [](const char *path_ptr, tiledb_object_t obj,
+                        void *pyfunc) {
+  std::string my_path(path_ptr);
+  std::string my_objtype;
+  if (obj == TILEDB_GROUP) {
+    my_objtype = "group";
+  }
+  if (obj == TILEDB_ARRAY) {
+    my_objtype = "array";
+  }
```

Comment:
What happens in the case of a `TILEDB_INVALID`?

Reply:
Would
```
else {
    return 0;
}
```
be ok?